### PR TITLE
Update MISCELLANEOUS.md

### LINKED ISSUE #6: Add emojis to MISCELLANEOUS.md
It would probably be helpful to have a list of server emojis in MISCELLANEOUS.md; ideally with names and images.

### PR DESCRIPTION
**Pull request by teamshortcut**
Added emojis. These are currently sorted in order of when they were added; if somebody would like to logically group them (e.g. Lyne ones together) they can, but I've spent enough time on this already. Resolves #6

### CODE REVIEW

File: MISCELLANEOUS.md [hunk 1]
```diff
@@ -1,3 +1,68 @@
...
+:thinkingeyes: | ![](https://cdn.discordapp.com/emojis/450068992770703360.png?v=1)
+:rolf: | ![](https://cdn.discordapp.com/emojis/450068992850264104.png?v=1)
+:this: | ![](https://cdn.discordapp.com/emojis/450070223278702592.png?v=1)
+:rip: | ![](https://cdn.discordapp.com/emojis/450070223660253184.png?v=1)
+:oof: | ![](https://cdn.discordapp.com/emojis/450084741421465633.png?v=1)
+:fingerguns: | ![](https://cdn.discordapp.com/emojis/450243028758822922.png?v=1)
+:quitethoughtful: | ![](https://cdn.discordapp.com/emojis/450657080995741712.png?v=1)
+:kek: | ![](https://cdn.discordapp.com/emojis/450657081180291074.png?v=1)
+:godacrossthepond: | ![](https://cdn.discordapp.com/emojis/451069985582743572.png?v=1)
+:blobFluffyTwitsHotChoccie: | ![](https://cdn.discordapp.com/emojis/451474199102619650.png?v=1)
```
Comment (thebeanogamer) [inline MISCELLANEOUS.md:38]: Why does this still exist?

File: MISCELLANEOUS.md [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+:OhRlyNow: | ![](https://cdn.discordapp.com/emojis/450068991210160129.png?v=1)
+:dab: | ![](https://cdn.discordapp.com/emojis/450068991222874114.png?v=1)
+:KMS: | ![](https://cdn.discordapp.com/emojis/450068991252103168.png?v=1)
+:smirk~1: | ![](https://cdn.discordapp.com/emojis/450068991579258880.png?v=1)
+:rollsafe: | ![](https://cdn.discordapp.com/emojis/450068991990300672.png?v=1)
+:thot: | ![](https://cdn.discordapp.com/emojis/450068992204210195.png?v=1)
+:NoU: | ![](https://cdn.discordapp.com/emojis/450068992284164106.png?v=1)
+:ummmmm: | ![](https://cdn.discordapp.com/emojis/450068992522977293.png?v=1)
+:thinkharder: | ![](https://cdn.discordapp.com/emojis/450068992594411521.png?v=1)
+:thonkharder: | ![](https://cdn.discordapp.com/emojis/450068992762183690.png?v=1)
```
Comment (thebeanogamer) [inline MISCELLANEOUS.md:28]: This is the same as :quitethoughtful:, so one can be removed.
https://github.com/CyberDiscovery/meta/blob/443faefc40514be9bb36923683661265046aa153/MISCELLANEOUS.md#L35

File: MISCELLANEOUS.md [hunk 3]
```diff
@@ -1,3 +1,68 @@
...
+:blobhmm: | ![](https://cdn.discordapp.com/emojis/450063616876609546.png?v=1)
+:blobhyperthink: | ![](https://cdn.discordapp.com/emojis/450063618306867200.png?v=1)
+:blobthonk: | ![](https://cdn.discordapp.com/emojis/450063618902458415.png?v=1)
+:reeeee: | ![](https://cdn.discordapp.com/emojis/450063621037228044.png?v=1)
+:hypers: | ![](https://cdn.discordapp.com/emojis/450063621385355265.png?v=1)
+:god: | ![](https://cdn.discordapp.com/emojis/450068990799380501.png?v=1)
+:OhRlyNow: | ![](https://cdn.discordapp.com/emojis/450068991210160129.png?v=1)
+:dab: | ![](https://cdn.discordapp.com/emojis/450068991222874114.png?v=1)
+:KMS: | ![](https://cdn.discordapp.com/emojis/450068991252103168.png?v=1)
+:smirk~1: | ![](https://cdn.discordapp.com/emojis/450068991579258880.png?v=1)
```
Comment (thebeanogamer) [inline MISCELLANEOUS.md:22]: Typo

**Review by thebeanogamer — CHANGES_REQUESTED** [review]
A couple small issues, plus the need for sorting.

File: MISCELLANEOUS.md (hunk 2 above)
Comment (thebeanogamer) [reply MISCELLANEOUS.md:28]: Correction: similar, but not the same:

Name | Picture
--- | ---
:quitethoughtful: | 
:thinkharder :|

Comment (thebeanogamer) [reply MISCELLANEOUS.md:28]: _I guess we can keep both if you really want, but it feels redundant_

**Comment by teamshortcut** [comment]
The change to the emoji name is incorrect (check the server), and the others are kind of irrelevant; this is making a list of current ones. Once this is merged, that would be the point for discussion on which ones to keep.

**Comment by thebeanogamer** [comment]
Unicode conflicting with server emoji makes me 😢 :
image

**Comment by thebeanogamer** [comment]
> 
> 
> The change to the emoji name is incorrect (check the server), and the others are kind of irrelevant; this is making a list of current ones. Once this is merged, that would be the point for discussion on which ones to keep.

If you'd prefer emoji issues be addressed further down the line, we can merge this now, it just seemed like a good chance to kill 2 birds with one stone.

**Review by thebeanogamer — APPROVED** [review]

**Comment by lightspeedana** [comment]
I see no big visible issues, minor stuff can be fixed later?

**Review by lightspeedana — APPROVED** [review]

**Pull request merged**